other-docs/guides/upgrading/v7.md

```diff
@@ -0,0 +1,141 @@
+- For Local Chassis run `composer chassis exec -- wp core update-db --network`
+- For Local Server run `composer local-server cli -- core update-db --network`
+
+### Elasticsearch Updates
+
+Altis v7 requires changes to the Elasticsearch mappings so you will need to reindex your content after the upgrade.
+
+- On [Altis Dashboard](https://dashboard.altis-dxp.com/) find the stack you have deployed to and run `elasticpress index --setup --network-wide --url=<host name>` (`wp` is automatically prepended) in the WP CLI tab
+- For Local Chassis run `composer chassis exec -- wp elasticpress index --setup --network-wide --url=<host name>`
+- For Local Server run `composer local-server cli -- elasticpress index --setup --network-wide`
+
+
+## Breaking Changes
+
+### PHP 7.4
+
+Altis v7 requires PHP 7.4 support. Both Local Server and Local Chassis now default to using PHP 7.4. You may need to update your application to run on PHP 7.4.
+
+The [full list of breaking changes in PHP 7.4 can be found here](https://www.php.net/manual/en/migration74.incompatible.php). We recommend the [PHP Compatibility standard for PHP Code Sniffer to automate finding issues that need fixing.](https://github.com/PHPCompatibility/PHPCompatibility)
+
+### ElasticPress 3.5.6
+
+The ElasticPress dependency has been updated from 3.4.3 to 3.5.6. This update requires reindexing your content. [A full list of ElasticPress changes for each release can be found here](https://github.com/10up/ElasticPress/releases).
+
+### Limit Login Attempts Removed
+
+The Limit Login Attempts feature has been removed in favour of rules to handle this at the infrastructure level. This avoids causing database writes during brute force attacks and subsequent performance problems.
+
+Any related configuration
+
+### S3 Uploads 3.0.0
+
+The latest version of S3 Uploads has been refactored to use namespaces instead of prefixed class names and functions.
+
+If you have any code referencing the class `S3_Uploads` this will need to be updated to `S3_Uploads\Plugin`. Additionally any other references to classes starting with `S3_Uploads` should be updated to use `S3_Uploads` as the namespace, for example `S3_Uploads_Local_Stream_Wrapper` becomes `S3_Uploads\Local_Stream_Wrapper`. Any functions prefixed with `s3_uploads_*` have also been moved into the `S3_Uploads` namespace, for example `s3_uploads_enabled()` is now `S3_Uploads\enabled()`.
+
+All constants remain unchanged.
+
+### Altis Experiments
+
+The experiments feature of native analytics has been merged into the main native analytics codebase. This means any references in your code to the namespace `Altis\Experiments` should be updated to `Altis\Analytics\Experiments`.
+
+Additionally the `Altis\Analytics\Experiments\register_goal()` function accepts a different set of arguments than it did in Altis v6.
```

Comment:
Could you summarize or link to that change?

Reply:
```suggestion
Additionally the `Altis\Analytics\Experiments\register_goal()` function accepts a different set of arguments than it did in Altis v6, [see the Goal Tracking documentation for full details](docs://analytics/native/goal-tracking/#php).
```